### ISSUE TITLE
Create a performance test suite

### ISSUE THREAD

**Issue by matozoid** [javaparser/javaparser#1021]
We're currently unable to track the performance impact of a new feature.

**Comment by ftomassetti** [comment]
Yes, this would be useful, but I would not know how to implement it correctly

**Comment by matozoid** [comment]
Hmmm, maybe the simplest way to start would be to take the unit test that currently parses all JP source code and time it.

**Comment by DeepSnowNeeL** [comment]
To do it right, we could use JMH
http://openjdk.java.net/projects/code-tools/jmh/ ?

**Comment by matozoid** [comment]
:-O I didn't that existed, looks cool!
